io.openems.edge.common/src/io/openems/edge/common/channel/doc/Level.java

```diff
@@ -3,16 +3,32 @@
 /**
  * Severity/visibility Level
  */
-public enum Level {
-	INFO(1), WARNING(2), FAULT(3);
-	
+public enum Level implements OptionsEnum {
+	OK(0, "Ok"), //
```

Comment:
What is thje difference between OK and Info?

Reply:
Good point. I added the following javadoc:
- "OK" indicates, that everything is OK and there are no messages.
- "Info" indicates, that everything is OK, but there is at least one informative messages available.

Differentiation is required for the UI. With "Info" it knows, that it needs to search trough the StateChannels to get the informative message.